molr-commons/src/main/java/io/molr/commons/domain/ExecutionStrategy.java

```diff
@@ -6,7 +6,7 @@
 
 public enum ExecutionStrategy {
 
-    PAUSE_ON_ERROR, PROCEED_ON_ERROR, ABORT_ON_ERROR;
+    PAUSE_ON_ERROR, PROCEED_ON_ERROR, ABORT_ON_ERROR, ABORT_ON_ERROR_LENIENT;
```

Comment:
What is the difference between 'ABORT_ON_ERROR' and 'ABORT_ON_ERROR_LENIENT'

... Some javadoc would be useful here ...

And potentially in some md file somehow a table like

ExecutionStrategy vs BlockAttributes (and mayjor events), which explains the effect in all combinations.